build-output-api/on-demand-isr-revalidation/.vercel/output/functions/revalidate.func/index.js

```diff
@@ -0,0 +1,37 @@
+const https = require('https');
+
+// The bypass token can be a randomly generated string of at least 32 characters.
+// This is meant to be *private* - DO NOT expose this value on the client-side.
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
+
+module.exports = (req, res) => {
+  const proto = req.headers['x-forwarded-proto'];
+  const host = req.headers.host;
+  const deployedUrl = `${proto}://${host}`;
+
+  const options = {
+    hostname: host,
+    port: 443,
+    path: '/',
+    method: 'GET', // MUST be "GET"; a "HEAD" request will not work
+    headers: {
+      'x-prerender-revalidate': bypassToken
+    }
+  };
+  const revalidateRequest = https.request(options, revalidateResponse => {
+    res.setHeader('Content-Type', 'text/html; charset=utf-8')
```

Comment:
```suggestion
    res.setHeader('Content-Type', 'text/html; charset=utf-8')
    
    if (revalidateResponse.headers['x-vercel-cache'] !== 'REVALIDATED') {
      res.statusCode = 500
      res.end(`Failed to revalidate`
    }
```
Might be good to explain how they can check if the revalidation was successful using the cache header as a successful revalidation can occur for non-200 status codes.